#! /bin/sh -e
## 05_qvfb_cpp.dpatch by Simon Hausmann <simon@kde.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix qvfb to obey our standards

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

diff -ruN qt-x11-free-3.1.1+cvs.20030123-old/tools/qvfb/pda.skin qt-x11-free-3.1.1+cvs.20030123/tools/qvfb/pda.skin
--- qt-x11-free-3.1.1+cvs.20030123-old/tools/qvfb/pda.skin	2002-09-06 14:32:46.000000000 +0200
+++ qt-x11-free-3.1.1+cvs.20030123/tools/qvfb/pda.skin	2003-01-26 20:46:30.000000000 +0100
@@ -1,4 +1,4 @@
-pda_up.png pda_down.png
+/usr/share/qvfb/pda_up.png /usr/share/qvfb/pda_down.png
 57 81
 240 320
 0.5
diff -ruN qt-x11-free-3.1.1+cvs.20030123-old/tools/qvfb/qvfb.cpp qt-x11-free-3.1.1+cvs.20030123/tools/qvfb/qvfb.cpp
--- qt-x11-free-3.1.1+cvs.20030123-old/tools/qvfb/qvfb.cpp	2002-10-04 17:27:31.000000000 +0200
+++ qt-x11-free-3.1.1+cvs.20030123/tools/qvfb/qvfb.cpp	2003-01-26 20:47:19.000000000 +0100
@@ -80,7 +80,7 @@
 	bool vis = isVisible();
 	if ( vis ) hide();
 	menuBar()->hide();
-	Skin *skin = new Skin( this, skin_name, w, h );
+	Skin *skin = new Skin( this, QString::fromLatin1( "/etc/qt3/qvfb/" ) + skin_name, w, h );
 	view = new QVFbView( display_id, w, h, d, skin );
 	skin->setView( view );
 	view->setMargin( 0 );
